functions-methods.js

```diff
@@ -19,8 +29,29 @@
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
+const email4 = "novi.nlaapjesk@outlook.com"
 
+function typeOfEmail(email){
+    let emailType = " ";
+if (email.includes("novi-education.nl")){
+    emailType = "Student"}
+else if (email.includes("outlook.com")) {
+    emailType = "Extern"}
+else if (email.includes("outlook.com") && email.includes("novi.nl") ) {
+    emailType = "Extern"}
+// is dit de juiste oplossing?
+else if (email.includes("novi.nl")) {
+    emailType = "Medewerker"}
+else {
+    emailType = "onbekend"
+}
+    return emailType
```

Comment:
Je zou hier ook een switch voor kunnen gebruiken. In elke case zou je dan direct de waarde terug kunnen geven in plaats van eerst een variabele van een waarde te voorzien en dan pas terug te geven